controllers/TodoController.js

```diff
@@ -0,0 +1,118 @@
+const {Todo} = require('../models')
+
+class TodoController {
+    static async create(req, res) {
+        const query = {
+            title: req.body.title,
+            description: req.body.description,
+            status: req.body.status == 'true' ? true : false,
+            due_date: req.body.due_date
+        }
+
+        try {
+            const todo = await Todo.create(query)
+            res.status(201).json(todo)
+        }
+
+        catch(err) {
+            if (err.name == 'SequelizeValidationError') {
+                let errors = []
+                err.errors.forEach(element => {
+                    errors.push(element.message)
+                });
+                res.status(400).json({message: errors.join()})
+            } else {
+                res.status(500).json('Internal server error')
+            }
+        }
+    }
+
+    static async show(req, res) {
+        try {
+            const todo = await Todo.findAll()
+
+            res.status(200).json(todo)
+        }
+        catch(err) {
+            res.status(500).json.json('Internal server error')
+        }
+    }
+
+    static async showById(req, res) {
+        try {
+            const todo = await Todo.findOne({
+                where: {
+                    id: req.params.id
+                }
+            })
+            // console.log(todo)
+            if (todo == null) {
+                res.status(404).json({message: 'not found'})
+            } else {
+                res.status(200).json(todo)
+            }
+        }
+
+        catch(err) {
+            res.status(500)
+        }
+    }
+
+    static async update(req, res) {
+        const query = {
+            title: req.body.title,
+            description: req.body.description,
+            status: req.body.status == 'true' ? true : false,
```

Comment:
BIsa juga dibuat seperti ini

```suggestion
            status: Boolean(req.body.status) // di convert jadi boolean,
```